components/ca-form.js

```diff
@@ -333,8 +333,8 @@ define([
 
                 // erase current form
                 clearElement(this.form);
-                this.setAttribute('create-url', '');
-                this.setAttribute('update-url', '');
+                this.removeAttribute('create-url');
```

Comment:
Well, as per @tapina's comment, I should be removing the attribute rather than setting it to a zero-length string.  there is no utility method for that, only getting and setting.

Secondly, these utility methods contain one line of code - setAttribute and getAttribute respectively.  Why why why are we using utility methods to obfuscate the (almost identical) native method?  I don't understand the logic of that, at all!